src/handlers/meta_transaction_handlers.ts

```diff
@@ -189,6 +189,15 @@ function parseRequestParams(req: express.Request): MetaTransactionQuoteRequestPa
             },
         ]);
     }
+    if (slippagePercentage < 0.001) {
```

Comment:
nit: Set this [config.ts](https://github.com/0xProject/0x-api/blob/master/src/config.ts#L368) to live with other config constants. 

Something like `META_TX_MIN_SLIPPAGE_ALLOWED`. Probably doesn't need to be ENV var configurable,.